src/Pages/License/Generate/OptionalDetails.jsx

```diff
@@ -0,0 +1,104 @@
+import React from "react";
```

Comment:
Separate groups for react, material ui and internal components. Put line breaks between each group

Reply:
done